aio/tools/transforms/config.js

```diff
@@ -1,8 +1,8 @@
 const { resolve } = require('path');
 const { readdirSync } = require('fs');
 
-const BAZEL_OUTPUT_PATH = process.env.BAZEL_DGENI_OUTPUT_PATH;
-const PROJECT_ROOT = resolve(__dirname, '../../..');
+const BAZEL_OUTPUT_PATH = process.env.BAZEL_DGENI_OUTPUT_PATH || "dist";
+const PROJECT_ROOT = resolve(process.env.PROJECT_ROOT || ".");
```

Comment:
Nit:

```suggestion
const PROJECT_ROOT = resolve(process.env.PROJECT_ROOT || '.');
```

So, what does `.` resolve to?

Reply:
The current working directory. The way I have this running under bazel, the cwd of the dgeni node process always runs in the execroot (project root).